hrjobcontract/js/src/job-contract/controllers/contract-list.js

```diff
@@ -124,53 +129,51 @@ define([
         $scope.contractPast = $filter('orderBy')($scope.contractPast, '-is_primary');
       };
 
-      $scope.modalContract = function(action) {
-
+      $scope.modalContract = function (action) {
         if (!action || action !== 'new') {
           return null;
         }
 
-        var modalInstance,
-          options = {
-            appendTo: $rootElement.find('div').eq(0),
-            templateUrl: settings.pathApp + 'views/modalForm.html?v=2222',
-            size: 'lg',
-            controller: 'ModalContractNewCtrl',
-            windowClass: 'modal-contract',
-            resolve: {
-              model: function() {
-                return $scope.model;
-              },
-              utils: function() {
-                return $q.all(angular.extend(promiseUtils, {
-                  contractListLen: $scope.utils.contractListLen
-                }));
-              }
+        var modalInstance;
+        var options = {
+          appendTo: $rootElement.find('div').eq(0),
+          templateUrl: settings.pathApp + 'views/modalForm.html?v=2222',
+          size: 'lg',
+          controller: 'ModalContractNewCtrl',
+          windowClass: 'modal-contract',
+          resolve: {
+            model: function () {
+              return $scope.model;
+            },
+            utils: function () {
+              return $q.all(angular.extend(promiseUtils, {
+                contractListLen: $scope.utils.contractListLen
+              }));
             }
           }
+        };
 
         modalInstance = $modal.open(options);
 
-        modalInstance.result.then(function(contract) {
-          ContractService.updateHeaderInfo(); +
-          contract.is_current ? $scope.contractCurrent.push(contract) : $scope.contractPast.push(contract);
+        modalInstance.result.then(function (contract) {
+          ContractService.updateHeaderInfo();
+          +contract.is_current ? $scope.contractCurrent.push(contract) : $scope.contractPast.push(contract);
 
           if (+contract.is_primary) {
             $scope.toggleIsPrimary(contract.id);
           }
 
           $window.location.assign(UtilsService.getManageEntitlementsPageURL(contract.contact_id));
         });
-      }
-
-      $scope.delete = function(contractId) {
+      };
 
-        function removeContractById(contractArray, id) {
-          var i = 0,
-            len = contractArray.length;
+      $scope.delete = function (contractId) {
```

Comment:
If it's not too troublesome, can you please make this file follow the style guide since you have already made considerable changes to it?

- Rename it to `contract-list.controller.js`.
- Use `vm` instead of `$scope` where it makes sense.
- Declare functions in alphabetical order, etc.

Reply:
I like you suggestion, but the `jobcontract` repo will be migrated to angular style guide very soon. And since, there will be many changes  if we considering adopting angular style guide in jobcontract in this PR.  I prefer to leave it for now.